src/routes/index.svelte

```diff
@@ -73,23 +50,82 @@
     let loading = false
     let highlightedSymbolIndex = null
 
+    let isDocumentReady = false
+
     onMount(() => {
+        let DEFAULT_OPTIONS = []
+
+        if (typeof document !== 'undefined') {
```

Comment:
Is there ever an instance where document is undefined within `onMount`? I would've assumed not